Fails with `The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received an instance of Document`

I just upgraded to v2 and have started getting failures with:
`The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received an instance of Document`

For example, I run it (and it fails) like this:

```shell
$ export DEBUG=pin-github-action* 
$ ./node_modules/.bin/pin-github-action \
    --allow-empty \
    --comment ' {ref}' \
    --allow 'myorg/*' \
    --ignore-shas \
    .github/workflows/my-test-suite.yml
    
2025-01-09T14:25:33.759Z pin-github-action:extract-actions Processing workflow
2025-01-09T14:25:33.762Z pin-github-action:check-allowed-repos Skipping myorg/some-workflows due to pattern match in myorg/*
The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received an instance of Document
Error: Process completed with exit code 1.
```

--- a/bin.js
+++ b/bin.js
@@ -63,7 +63,7 @@ const packageDetails = JSON.parse(
         debug,
         comment
       );
-      fs.writeFileSync(filename, output.workflow);
+      fs.writeFileSync(filename, output.workflow.toString());
     }
 
     // Once run on a schedule, have it return a list of changes, along with SHA links

